### DISCUSSION TITLE
Remove Asset Manager

### DISCUSSION THREAD

**Discussion by jloguercio** [GrapesJS/grapesjs#4109] [Q&A]
There is a way to remove the double click show asset manager event from images ? i made another way to insert / change images.

Thanks

**Comment by artf** [comment]
You can extend the component image view and update their events:
https://github.com/artf/grapesjs/blob/0af135de7f625a36389dfb0bfff66c595d33dca1/src/dom_components/view/ComponentImageView.js#L8
